lib/Auth.js

```diff
@@ -118,19 +121,56 @@ class Auth {
     //
     //
     setUsernamePassword(username, password) {
+        this._useGoogleClientAuth = false;
         this._username = username;
         this._password = password;
     }
     //
     //
     //
+    setGoogleClientIdAndEmails(clientid, emails) {
+        this._useGoogleClientAuth = true;
+        this._googleClientId = clientid;
+        this._emails = emails;
+    }
+    //
+    //
+    //
+    useGoogleClientAuth() {
+        return this._useGoogleClientAuth;
+    }
+    //
+    //
+    //
+    getGoogleClientId() {
+        return this._googleClientId;
+    }
+    //
+    //
+    //
+    getGoogleClientEmails() {
+        return this._emails;
+    }
+    //
+    //
+    //
+    isGoogleClientEmailValid(email) {
+        if (this._useGoogleClientAuth) {
+            const emails = this._emails.split(';');
```

Comment:
The Readme says, it's a comma separated list. But here you split by semicolons.

Reply:
Sorry, my mistake. It was my first idea. I've fixed it.

Reply:
Sorry again, I've changed the locales but I forgot the README.